src/src/pages/topics/index.js

```diff
@@ -197,9 +200,26 @@ function Topics() {
                 </div>
               </div>
             </div>
+            ) : null}
+            
           </div>
         }
-      }
+      },
+      {
+        accessorKey: 'arrow',
+        header: 'arrow',
+        id: 'arrow',
+        size: '1',
+        enableColumnFilterModes: false,
+        muiTableBodyCellProps: {
+          align: 'center',
+        },
+        Cell: ({ cell}) => {
+          return <div>
+            {!cell.row.getIsExpanded() ? ( <ChevronDown /> ) : <ChevronUp />}
```

Comment:
I would reverse the logic something like: `cell.row.getIsExpanded() ? <ChevronUp />  : <ChevronDown />`